ftd/fpm.ftd

```diff
@@ -329,6 +329,7 @@ string list telegram:
 string list -telegram:
 string list github:
 string list -github:
+string list github-like:
```

Comment:
This list is not as per fpm-controller code. https://github.com/FifthTry/fpm-controller/blob/69291cf86fc70421f229f930f66c67069d6442e1/src/dj/child_auth/views.py#L360-L373.